openshift-ci/enable_frrk8s_on_cno.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
shellcheck enable_frrk8s_on_cno.sh is giving some improvements. Since this is a new script, I think it will be nice to comply with shellcheck

Reply:
yep, I'll do.